.github/workflows/test-smokes.yml

```diff
@@ -174,9 +174,24 @@ jobs:
         with:
           version: "1.11.3"
 
+      - name: Checkout dev branch of QuartoNotebookRunner
+        uses: actions/checkout@v4
+        with:
+          repository: PumasAI/QuartoNotebookRunner.jl
+          ref: jk/workers-status
+          path: qnr
+
+      - name: Set QUARTO_JULIA_PROJECT environment variable
+        run: echo "QUARTO_JULIA_PROJECT=$GITHUB_WORKSPACE/qnr" >> $GITHUB_ENV
+        shell: bash
+
```

Comment:
Do I understand correctly that this would eventually be changed to check out the `main` brtanch of QuartoNotebookRunner once that's released?

The question I have is: will our users have to check out this branch in order for the feature to work?

Reply:
the PR with those changes is waiting for review over there, given that you are positive about the changes made here, we will probably go ahead with that in a little bit (cc @MichaelHatherly). Then, we can simply change the version number in the Project.toml file like we usually do and replace this CI script modification.

Reply:
Yeah, I'll be looking at the backend PR this afternoon.

Reply:
Backend PR looks fine, @jkrumbiegel feel free to merge and tag a new version whenever you're ready.